### ISSUE TITLE
Figure out a way to better detect islands and route under all conditions.

### ISSUE THREAD

**Issue by xivk** [itinero/routing#104]
See issue #85 

> Something else needs to be figured out here, probably marking ways that are part of a routing island per profile to make things really rock solid. I'm going to open a new issue for this.

**Comment by xivk** [comment]
A possible solution could be once #112 is finished to detect islands (optionally) during preprocessing and store them in the routerdb.

**Comment by xivk** [comment]
This has been implemented too, available in 1.4, should become default from 2.0 on.
